packages/stark-build/config/webpack.common.js

```diff
@@ -457,18 +441,6 @@ module.exports = function(options) {
 				{}
 			),
 
-			/**
-			 * Plugin: HashedModuleIdsPlugin
-			 * Description: This plugin will cause hashes to be based on the relative path of the module,
```

Comment:
Why moving this plugin in prod config instead of keeping it for every mode (prod, dev)?